package.json

```diff
@@ -20,7 +20,10 @@
     "build:shared": "yarn workspace @gradejs-public/shared run build",
     "build:worker": "yarn workspace @gradejs-public/worker run build",
     "build:public-api": "yarn workspace @gradejs-public/public-api run build",
-    "build:backend": "yarn build:shared && yarn build:public-api && yarn build:worker"
+    "build:backend": "yarn build:shared && yarn build:public-api && yarn build:worker",
+    "dev": "bash cli/local_start.sh",
+    "worker:local:syncPackages": "curl -d '{\"type\":\"syncPackageIndex\"}' -H \"Content-Type: application/json\" -X POST http://localhost:8084",
```

Comment:
I would rather have this in the readme, instead of creating a separate command for each task

Reply:
It's just a shortcut. I'd prefer to have a working shortcut for each task than a readme entry (though, _both_ shortcut and readme would be ideal :) ) 

I'll rename it to clarify it's a dev task. If someday we have too many dev tasks in package.json and it will get us nervous, we always can move them to separate bash script or something.

Reply:
Good point, however the logic might break once we start adding worker tasks with custom payloads. 
By having a command per worker task means that we also should update the `package.json` file with all affected changes, which seems redundant for the monorepo

Reply:
I don't think this is redundant, as it's a tool for the developer's convenience. Let's leave it here for now.